site/content/en/docs/reference/labels-and-annotations.md

```diff
@@ -116,6 +116,15 @@ Used on: [Plain Pods](/docs/tasks/run/plain_pods/).
 
 The annotation key is used to indicate how many Pods to expect in the group.
 
+### `kueue.x-k8s.io/pod-group-pod-index
+
+Type: Label
+
+Example: ``kueue.x-k8s.io/pod-group-pod-index: "0"`
```

Comment:
The code format does not seem to be correct.

```suggestion
Example: `kueue.x-k8s.io/pod-group-pod-index: "0"`
```

I fixed it in https://github.com/kubernetes-sigs/kueue/pull/3682